### PR TITLE
updates in microphone from April

### PR DESCRIPTION
**Pull request by karandeabhijitsudam**

### CODE REVIEW

**Comment by karandeabhijitsudam** [comment]
resolved conflicts

File: ressources/PolyServ
Comment (Victor-Haefner) [inline ressources/PolyServ:1]: Why do we need it here?
please remove if possible

File: examples/tmp2.jpeg
Comment (Victor-Haefner) [inline examples/tmp2.jpeg:1]: please remove

File: ressources/PolyServ
Comment (karandeabhijitsudam) [reply ressources/PolyServ:1]: Removed!

File: examples/tmp2.jpeg
Comment (karandeabhijitsudam) [reply examples/tmp2.jpeg:1]: removed!

**Pull request merged**